src/main/java/org/kainos/ea/cli/Role.java

```diff
@@ -0,0 +1,35 @@
+package org.kainos.ea.cli;
+
+public enum Role {
+
+    ADMIN(1, "Admin"),
```

Comment:
How are we handling roles, do we have a DB table of the roles or are we using a magic number

Reply:
Replaced with https://github.com/anthony-raybould/bham1-back-end/pull/9/files#diff-d61e1278ed1c721e92809a0b4eece6c4b531f163a915224317e36e64eda034c8